CONTRIBUTING.md

```diff
@@ -1,11 +1,91 @@
+
+## Submitting Issues
+
+* Before creating a new issue, perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Aibm-watson+) to see if a similar issue has already been submitted.
+* You can create an issue [here](https://github.com/IBM-Watson/design-library/issues). Please include as many details as possible in your report, and consider using our [Emoji cheatsheet](#emoji-cheatsheet) to categorize your issue.
+* Please include the version of the Design Library you are using/viewing
+* When submitting a browser bug, please include the browser, version, operating system, and operating system version.
+* Include screenshots and animated GIFs whenever possible; they are immensely helpful.
+
+## Pull Requests
+
+* **DO NOT ISSUE A PULL REQUEST WITHOUT FIRST [SUBMITTING AN ISSUE](#submitting-issues)**
+* Pull requests should reference their related issues. If the pull request closes an issue, [please reference its closing in your commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/). Pull requests not referencing any issues will be closed.
+* Update the `CHANGELOG` with the changes made by your pull request, making sure to use the proper [Emoji](#emoji-cheatsheet).
+* Make sure you have [installed the development environment](https://github.com/IBM-Watson/design-library#installation), [updated your runner to the latest version](https://github.com/IBM-Watson/design-library#updating-the-runner), and have [run the library locally](https://github.com/IBM-Watson/design-library#running-locally) to ensure that your code works properly.
+* Follow our JavaScript and CSS styleguides. We have linters set up to catch most of it.
+* Ensure that you have [EditorConfig](http://editorconfig.org/) installed in your editor of choice and that it is functioning properly.
+* Do not squash or rebase your commits when submitting a Pull Request. It makes it much harder to follow your work and make incremental changes.
+* Update the [CHANGELOG](#maintaining-thechangelog) with your changes.
+* Ensure no Emoji tags are used in the title of your Pull Request
+
+### Git Commit Messages
+
+* Use the present tense (`"Add feature"` not `"Added Feature"`)
+* Use the imperative mood (`"Move cursor to…"` not `"Moves cursor to…"`)
+* Limit the first line to 72 characters or less
+* Include relevant Emoji from our [Emoji cheatsheet](#emoji-cheatsheet)
```

Comment:
Maybe change to: "Include relevant Emoji at the **start** of your commit." You mention it below in the emoji section but not here.